app/src/sync/process-initialization.ts

```diff
@@ -69,6 +59,35 @@ export async function update_directory() {
   ensureActiveProjects();
 }
 
+/**
+ * Create and store a listings object for this conductor instance
+ *
+ * @param url - URL of the Conductor instance
+ * @returns A Listings object
```

Comment:
This is where that shared interface would come in handy?

Reply:
done